krish98sai/geo-puzzle
# Add devise

To test:

1. git checkout add-devise

2. docker-compose build

3. docker-compose run web rails db:create

4. docker-compose run web rails db:migrate

5. docker-compose up -d

6. 192.168.99.100/users/sign_in

7. Make sure everything looks fine.
